src/category_theory/adjunction/limits.lean

```diff
@@ -95,11 +97,13 @@ def left_adjoint_preserves_colimits : preserves_colimits F :=
 omit adj
 
 @[priority 100] -- see Note [lower instance priority]
-instance is_equivalence_preserves_colimits (E : C ⥤ D) [is_equivalence E] : preserves_colimits E :=
+instance is_equivalence_preserves_colimits (E : C ⥤ D) [is_equivalence E] :
+  preserves_colimits_of_size E :=
 left_adjoint_preserves_colimits E.adjunction
```

Comment:
```suggestion
instance is_equivalence_preserves_colimits (E : C ⥤ D) [is_equivalence E] :
  preserves_colimits_of_size.{v u} E :=
left_adjoint_preserves_colimits E.adjunction
```


I think it would be better to specify the first two universe argument of `preserves_colimits_of_size` explicitly, and adjust the order a bit, so that we may write `is_equivalence_preserves_colimits.{w w'}` when we want a `preserves_colimits_of_size.{w w'}`.

The same for other `of_size` stuff.